mir_manipulation/mir_gripper_controller/CMakeLists.txt

```diff
@@ -50,23 +32,14 @@ endif()
 
 
 ### INSTALL ###
-install(PROGRAMS
-  ros/scripts/gripper_command_controller
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-)
+#install(PROGRAMS
```

Comment:
shouldn't you install `ros/scripts/teensy_gripper_controller_ros`? I assume it is executable